make-release.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+if [ -z "$1" ]; then
+    echo "Usage: $0 <release_version>"
+    exit 1
+fi
+
+RELEASE_VERSION=$1
+
+# Update CHANGES.md
+sed -i "s/## Unreleased/## Unreleased\n\n## $RELEASE_VERSION\n/" CHANGES.md
+echo "This description should answer the \"who needs this update?\" question."
+sleep 10
+${EDITOR:-vim} +7 CHANGES.md
```

Comment:
How about writing guide line in the CHANGES.md file itself? No need to sleep this way.

You can make something to make the line stand out, to avoid being forgotten.